src/Nox.Generator/Application/Dto/EntityDto.template.cs

```diff
@@ -17,8 +21,10 @@ namespace {{codeGeneratorState.ApplicationNameSpace}}.Dto;
 /// {{entity.Description}}.
 /// </summary>
 [AutoMap(typeof({{entity.Name}}CreateDto))]
+[PrimaryKey({{primaryKeyParams}})]
```

Comment:
this change should not be necessary, the model and the primary keys are being set by the Databaseconfiguration

I would expect changes only on the controller level

Reply:
Unfortunately, the following exception is thrown on any API call if this line of code is removed:

{
  "statusCode": 500,
  "Message": "The entity type 'MultipleIdsTypeDto' has multiple properties with the [Key] attribute. Composite primary keys configured by placing the [PrimaryKey] attribute on the entity type class, or by using 'HasKey' in 'OnModelCreating'."
} 

Even considering that we have HasKey, it still doesn't work without PrimaryKey

Reply:
As discussed, updated to HasKey